sirepo/package_data/static/js/light.js

```diff
@@ -253,7 +253,7 @@ app.directive('pageHeading', function() {
         template: `
             <nav class="navbar navbar-default navbar-static-top">
               <div class="container-fluid">
-                <a class="navbar-brand" href="/en/xray-beamlines.html">
+                <a class="navbar-brand" href="">
```

Comment:
@robnagler  When is this code used? I don't see code from light.js executing when I visit SRW.

Reply:
light.js is used at BNL. does href="" do the "right" thing?

Reply:
9847975a7cb50761b59da0f3bbff9d40064e7526